src/Servers/Kestrel/samples/http2cat/PseudoHeaderNames.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Http2cat's code lives over in https://github.com/dotnet/aspnetcore/tree/76ef7a069666e28c8123fa21b70c274191fb23d5/src/Shared/Http2cat